README.md

```diff
@@ -25,11 +25,28 @@ defp app_list(_),  do: app_list
 defp app_list,  do: [:logger]
 ```
 
+## Using with Phoenix and Ecto
+
+There is nothing special you need to do with Phoenix unless you decide to
+`import` your factory module.
+
+By default Phoenix `import`s `Ecto.Model` in the generated `ConnCase` and
+`ModelCase`  modules (found in `test/support/conn_case.ex` and
+`test/support/model_case.ex`). To import your factory we recommend excluding
+`build/2` or aliasing your factory instead.
```

Comment:
I'm not sure that adds value. To me it makes the library look incomplete and shady, without giving me any valuable information IMO :S

I'm leaning toward updating the README when we have a solution and just leaving this as-is in the meantime. What do you think?